webpack.js

```diff
@@ -87,4 +88,17 @@ webpackConfig.plugins.push(
 	})
 )
 
+webpackConfig.plugins = [
+	...webpackConfig.plugins,
+	// Generate reuse license files
+	new WebpackSPDXPlugin({
+		override: {
+			// TODO: Remove if they fixed the license in the package.json
+			'@nextcloud/axios': 'GPL-3.0-or-later',
+			'@nextcloud/vue': 'AGPL-3.0-or-later',
+			'nextcloud-vue-collections': 'AGPL-3.0-or-later',
+		}
+	}),
+]
+
 module.exports = webpackConfig
```

Comment:
still needs to disable the extract comments part of the terser minifier to remove the `.license.txt` files

Reply:
👍
Yes, just need to find the config snippet again 😅

Reply:
found it 😃